release.csx

```diff
@@ -0,0 +1,84 @@
+#r ".\packages\Octokit.0.26.0\lib\net45\Octokit.dll"
+#r ".\packages\system.runtime.interopservices.runtimeinformation\4.0.0\lib\net45\System.Runtime.InteropServices.RuntimeInformation.dll"
+
+using Octokit;
+
+const string repoOwner = "FakeItEasy";
+const string repoName = "FakeItEasy";
+const string remote = "upstream";
+
+var version = Args.FirstOrDefault();
+if (version == null)
+{
+    throw new Exception("No version supplied");
+}
+
+var releaseBranchName = $"release/{version}";
+
+var token = ReadTokenFromNetrc();
+
+var gitHubClient = new GitHubClient(new ProductHeaderValue("FakeItEasy-build-scripts"));
+gitHubClient.Credentials = new Credentials(token);

```

Comment:
I'm assuming the ,netrc contains a personal token, not username password, and don't particularly want to encourage username password anyhow.